packages/components/toast/toast.component.ts

```diff
@@ -47,9 +48,13 @@ let id = 0;
     },
     animations: [kbqToastAnimations.toastState],
     changeDetection: ChangeDetectionStrategy.OnPush,
-    encapsulation: ViewEncapsulation.None
+    encapsulation: ViewEncapsulation.None,
+    providers: [
+        { provide: KBQ_TITLE_TEXT_REF, useExisting: KbqToastComponent }]
 })
 export class KbqToastComponent implements OnDestroy {
+    @ViewChild('kbqTitleText', { static: false }) textElement: ElementRef;
```

Comment:
чтобы public-api не раздувать
```suggestion
    @ViewChild('kbqTitleText', { static: false }) private readonly textElement: ElementRef;
```

Reply:
а для чего он вообще?

Reply:
для обращения внутри html компонента)

Reply:
@artembelik проброс свойства директивы для селекта контейнера, который будет проверяться на overflow и в случае прохождения по условию, будет выводить тултип